web/client/components/TOC/Toolbar.jsx

```diff
@@ -330,14 +341,14 @@ class Toolbar extends React.Component {
                         </OverlayTrigger>
                         : null}
                     {this.props.activateTool.activateSwipeOnLayer && (status === 'LAYER') &&
-                        <OverlayTrigger
-                            key="layerSwipe"
-                            placement="top"
-                            overlay={<Tooltip id="layer-tooltip-swipe"><Message msgId="toc.compareTool" /></Tooltip>}>
-                            <Button key="layer-swipe" bsStyle={this.props?.swipeSettings?.active ? "success" : "primary"} className="square-button-md" onClick={() => this.showSwipeSettings(status)}>
-                                <Glyphicon glyph="transfer" />
-                            </Button>
-                        </OverlayTrigger>}
+                        <DropdownButtonT
```

Comment:
Can you try to include this with the diff I provided to you for toolbar ? I'd like to start to separate and reduce dependencies between plugin as much as possible, so we can make also the other buttons pluggable, in the near future?